codegen/src/api/mod.rs

```diff
@@ -324,9 +322,14 @@ impl RuntimeGenerator {
             })
             .collect();
 
+        let rust_items: Vec<_> = item_mod_ir.rust_items().collect();
```

Comment:
```suggestion
        let rust_items = item_mod_ir.rust_items();
```
I think `quote!` can handle the iterator directly?

Reply:
Oh, it does! Good catch, thanks